create_folder_structure.py

```diff
@@ -0,0 +1,28 @@
+import os
+import sys
+
+create_fols = [
```

Comment:
Since this is a global variable for the module, the general style is to use ALL CAPS for the name.

Reply:
@bltravis which module?